lib/lightning/projects.ex

```diff
@@ -245,6 +245,20 @@ defmodule Lightning.Projects do
     end)
   end
 
+  @spec project_retention_policy_for(Attempt.t()) ::
+          :retain_all | :erase_all | :erase_successful_only
+  def project_retention_policy_for(%Attempt{work_order_id: wo_id}) do
```

Comment:
note that we need to change this name so it's not confused with `retention_period`